build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfoes.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: ipblocksinfoes.crd.nsx.vmware.com
```

Comment:
`ipblockinfos`, not `infoes`. Use the same as `NetworkInfos`.

https://github.com/vmware-tanzu/nsx-operator/blob/7e0f497ce6aab612c57a3302e73375fcb18c1003/build/yaml/crd/vpc/crd.nsx.vmware.com_networkinfos.yaml#L7

Same for the file name.

Reply:
Updated, thanks!